dev/io.openliberty.mcp/src/io/openliberty/mcp/content/Content.java

```diff
@@ -86,6 +91,13 @@ default String getType() {
         return type().toString().toLowerCase();
     }
 
+    /**
+     * @param audience (may be {@code null})
+     * @param lastModified (may be {@code null})
+     * @param priority (may be {@code null})
+     */
+    public record Annotations(Role audience, String lastModified, Double priority) {}
```

Comment:
I'm not sure why `lastModified` isn't an `Instant`, but we'll leave it matching quarkus for the moment.